packages/next/src/client/components/react-dev-overlay/internal/container/RuntimeError/index.tsx

```diff
@@ -132,29 +132,23 @@ export const styles = css`
     padding: 0;
     border: none;
     background: none;
-    outline: none;
```

Comment:
Removing this will trigger a flash of the outline border and pending is still showing "cursor", can we still keep them?

Reply:
I'll change it to only show on focus-visible. Though the scenario you described is what `outline` is intended to do.